fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -902,6 +902,11 @@ protected void runAfterCatalogReady() {
         }
     }
 
+    @VisibleForTesting
+    public synchronized boolean isContainedInidToRecycleTime(long id) {
```

Comment:
```suggestion
    synchronized boolean isContainedInidToRecycleTime(long id) {
```

remove the access identifier, so the interface can be accessible from the same package, but not visible outside the package, which can be a workaround to allow UT accessing.

Reply:
Updated